node/node.go

```diff
@@ -1460,20 +1460,18 @@ func (n *Node) initHealthAPI() error {
 		availableDiskPercentage := n.resourceTracker.DiskTracker().AvailableDiskPercentage()
 
 		var diskSpaceErrors []error
-		if availableDiskBytes < n.Config.RequiredAvailableDiskSpace {
+
```

Comment:
A bit pedantic, but we don't need a slice anymore. There is only ever at most 1 error appended to the slice.